app/src/main/java/mozilla/lockbox/extensions/ViewNode+.kt

```diff
@@ -15,21 +17,21 @@ private fun AssistStructure.ViewNode.dumpNode(sb: StringBuilder = StringBuilder(
     val name = htmlInfo?.tag ?: className?.split('.')?.last() ?: "unknown"
 
     var attrs = listOf(
-        Pair("idEntry", idEntry ?: ""),
-        Pair("idPackage", idPackage ?: ""),
-        Pair("idType", idType ?: ""),
-        Pair("webDomain", webDomain ?: ""),
-        Pair("hint", hint ?: ""),
-        Pair("autofillValue", autofillValue?.textValue ?: ""),
```

Comment:
Is this the line that is throwing the exception?

Looking at the implementation of `getTextValue()` it looks like we should be testing `autofillValue?.isText()` before calling `.textValue`.

Perhaps:
```kotlin
"autofillValue" to autofillValue?.let { if (it.isText()) it.textValue } ?: emptyString
```

Reply:
That is what I'm doing in 
```
Pair("autofillValue", autofillValue?.isText ?: autofillValue?.textValue ?: emptyString)
```

I can change the syntax to use `let` instead of `?:`s if that's preferred